jest.config.js

```diff
@@ -1,20 +0,0 @@
-// For a detailed explanation regarding each configuration property, visit:
-// https://jestjs.io/docs/en/configuration.html
-
-module.exports = {
-  errorOnDeprecated: true,
-  notify: true,
-  preset: 'jest-puppeteer',
-  transform: {
-    '\\.js$': 'babel-jest',
-  },
-  testMatch: [
-    'ONLY_VISUAL_REGRESSION_TEST' in process.env
-      ? '**/test/screenshot.test.js'
-      : '**/test/**/*.test.js',
-  ],
```

Comment:
Need to find a way to maintain this behavior (we split Visual Regression to a separate job in CircleCI).

Reply:
Restored in 7c3b564.